holidays/calendars/gregorian.py

```diff
@@ -30,6 +29,14 @@
 WEEKDAYS = {w: i for i, w in enumerate(("mon", "tue", "wed", "thu", "fri", "sat", "sun"))}
 
 
+def _delta_days(dt: date, n: int) -> date:
+    """
+    Return date that is n days after (n > 0) or before (n < 0) specified date.
+    """
+
+    return date.fromordinal(dt.toordinal() + n)
```

Comment:
Let's add some flexibility to this in case we ever need something more than `days`:

```suggestion
def _timedelta(dt: date, days: int = 0) -> date:
    """
    Return date that is `days` days after (days > 0) or before (days < 0) specified date.
    """

    return date.fromordinal(dt.toordinal() + days)
```

Reply:
Done.